pkg/cmd/pr/view/view_test.go

```diff
@@ -564,18 +563,14 @@ func TestPRView_web_currentBranch(t *testing.T) {
 	defer http.Verify(t)
 	http.Register(httpmock.GraphQL(`query PullRequestForBranch\b`), httpmock.FileResponse("./fixtures/prView.json"))
 
-	var seenCmd *exec.Cmd
-	//nolint:staticcheck // SA1019 TODO: rewrite to use run.Stub
-	restoreCmd := run.SetPrepareCmd(func(cmd *exec.Cmd) run.Runnable {
-		switch strings.Join(cmd.Args, " ") {
-		case `git config --get-regexp ^branch\.blueberries\.(remote|merge)$`:
-			return &test.OutputStub{}
-		default:
-			seenCmd = cmd
-			return &test.OutputStub{}
-		}
+	cs, cmdTeardown := run.Stub()
+	defer cmdTeardown(t)
+
+	cs.Register(``, 0, "")
```

Comment:
An empty pattern will match anything. Please provide something about the command as pattern. Maybe we should have `Register()` panic if an empty pattern was passed

Reply:
This has been corrected. I did not mean to leave that empty pattern in there. It seems reasonable to have `Register()` panic if an empty pattern is passed to enforce test quality. Did you want me to make that change in this PR as well? Or would you like a separate PR for that?